diagnostic.md

```diff
@@ -21,13 +21,20 @@ Write the `curl` request you'd use to retrieve a list of all donuts on the
 server.
 
 ```sh
-# your answer here
+curl --request GET http://www.example.com/donuts
 ```
 
 Write an AJAX request to retrieve a list of all donuts on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = function() {
+  $.ajax({
```

Comment:
it's always a good idea to prevent your default actions so something like this:
```
let getDonuts = function (event) {
  event.preventDefault();
```